### DISCUSSION TITLE
aac Radio streams  do not work .Solved!

### DISCUSSION THREAD

**Discussion by hjfisch** [rern/rAudio#658] [Fixed Issues]
aac streams like Radio Swiss Jazz won't work

**Comment by klassikmann** [comment]
Settings > Player > FFmpeg checked?

**Comment by hjfisch** [comment]
Checked , but no luck.

**Reply by klassikmann** [reply]
Don't use https://stream.srg-ssr.ch/rsj/aacp_96.m3u. Instead use the content of the m3u:
http://stream.srg-ssr.ch/m/rsj/aacp_96

More luck?

**Reply by hjfisch** [reply]
Working now , thank's a lot

**Reply by phreatic62** [reply]
I found this with sources from Global, you have to look around for alternates.

**Reply by klassikmann** [reply]
@phreatic62 
What do you mean?

**Reply by phreatic62** [reply]
Some UK (and European?) radio stations are all part of Global, so from a PC you access them via the Global website, however if you copy the browser link for a station into rAudio, it won't work, so I had to look around for alternate streams.

e.g. https://www.globalplayer.com/live/lbc/uk/ does not work so I use http://media-ice.musicradio.com/lbcuk

**Reply by klassikmann** [reply]
You have to find the native streams by the webradio station to integrate the station in rAudio. The links provided by webradio aggregators like "radio.net" or "tunein.com" are often (mostly?) not suitable outside these commercial providers. In this thread native streams had been discussed.

**Reply by phreatic62** [reply]
Ironically, I think 'media-ice' is the aggregator and and 'global' is the native...but I'm not sure ;)

**Comment by klassikmann** [comment]
For completeness the five (perhaps more?) internet stream services of Radio Swiss:

**Radio Swiss Pop**
http://stream.srg-ssr.ch/m/rsp/aacp_96

**Radio Swiss Jazz**
http://stream.srg-ssr.ch/m/rsj/aacp_96

**Radio Swiss Classic DE**
http://stream.srg-ssr.ch/m/rsc_de/aacp_96

**Radio Swiss Classic FR**
http://stream.srg-ssr.ch/m/rsc_fr/aacp_96

**Radio Swiss Classic IT**
http://stream.srg-ssr.ch/m/rsc_it/aacp_96

Logos are available at 
https://logos.fandom.com/wiki/Category:Radio_stations_in_Switzerland
